### PR TITLE
Create KS with disableEntitlment to avoid end user entitlment

### PR DESCRIPTION
**Pull request by yossipapi**

### CODE REVIEW

File: lib/BackendClient.js [hunk 1]
```diff
@@ -73,7 +73,7 @@ var BackendClient = (function(){
                     client.setSessionId(results);
                     deferred.resolve();
                 }
-            }, adminSecret, null, kalturaTypes.KalturaSessionType.ADMIN, partnerId, null, null);
+            }, adminSecret, null, kalturaTypes.KalturaSessionType.ADMIN, partnerId, null, "all:*,disableentitlement");
```
Comment (kalturaguy) [inline lib/BackendClient.js]: i think it's better to add this  into the config.template 
"backendClient" : {
    "adminSecret" :"admin secret",
    "partnerId" : 12345,
    "serviceUrl" : "http://serviceURL",
    "ksSessionRefreshIntervalMinutes" : 720,
    "ksPrivileges":  "all:*,disableentitlement"
  },

**Comment by yossipapi** [comment]
@kalturaguy updated with your suggestion please review.

**Comment by kalturaguy** [comment]
@yossipapi  Looks OK, but you need to update OLD liveDVR saas tag to match the new config

**Comment by yossipapi** [comment]
I know ... I see that the latest tag was released from master so should I do my changes on teh master branch?

**Pull request merged**